extensions/wikia/AdEngine/src/templates/big-fancy-ad-above-mobile-config.js

```diff
@@ -36,15 +36,19 @@ export const getConfig = mercuryListener => ({
 		this.adjustPadding(iframe, this.slotParams);
 		window.addEventListener('resize', onResize);
 
-		if (this.mercuryListener) {
-			this.mercuryListener.onPageChange(() => {
+		if (mercuryListener) {
+			mercuryListener.onPageChange(() => {
 				page.classList.remove('bfaa-template');
 				document.body.classList.remove('vuap-loaded');
 				document.body.classList.remove('has-bfaa');
 				document.body.style.paddingTop = '';
 				this.adsModule.setSiteHeadOffset(0);
 				window.removeEventListener('resize', onResize);
 			});
+			mercuryListener.onMenuOpenClear();
```

Comment:
We can move this method call into `mercuryListener`.

Reply:
Yes, we can! Moved.